### ISSUE TITLE
Wrong read ID output when using custom cDNA adapter mode

### ISSUE THREAD

**Issue by obegik** [adnaniazi/tailfindr#24]
Hi Adnan,

I came across with a problem when I used custom cDNA adapter mode. In first column, I realised that some read IDs contained an extra "read" text, which resulted in invalid estimation. 

Thanks,
Oguzhan

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/46004829/153301843-95f3e457-76b9-44d7-acd3-3edc66fd0243.png">

**Comment by adnaniazi** [comment]
Hi Oguzgan,

Thank you for using tailfindr. 

Tailfindr expects you to provide it either single-read Fast5 files

**Comment by obegik** [comment]
Hi Adnan,

Thanks for your prompt reply. I actually doubt that this is the issue because when I ran tailfindR before with default settings (without providing the adapter sequence), I did not have this issue. This only happened when I introduced the adapter sequence. 

Thanks,
Oguzhan

**Comment by adnaniazi** [comment]
Can you email me on adnaniazi@gmail.com some example data and the command that you are using. I will try to debug it.

**Comment by adnaniazi** [comment]
I think the issue is as I described above in my earlier comment. I designed tailfindr in a way that it first detects the type of the Fast5 file (single-read or multiread) by reading just one random read from all the available FAST5 files in the top-level dir, and then depending filetype detected (single or multi-read), dispatching all the reads to either single-read or multi-read processing scripts. Now if you are providing a mix of single and multi-read files, tailfindr will only work for one and fail completely for the other type. This is what is happening in your case, in all likelihood. So just rerun tailfindr separately on a folder that contains single fast5 files, and then separately on another folder containing multi-read FAST5 files, and see what happen.
